### DISCUSSION TITLE
How to use SkyAPM.Agent.AspNet to monitor asp.net application（.net framewolk 4.6.1）

### DISCUSSION THREAD

**Discussion by StevenWangGitHub** [SkyAPM/SkyAPM-dotnet#546] [Q&A]
We are having application which is asp.net application（.net framewolk 4.6.1）on IIS. How to use SkyAPM.Agent.AspNet to monitor. Like  this url not to use https://www.cnblogs.com/yjrl/p/12460265.html

**Comment by liuhaoyang** [comment]
支持ASP.NET的最后一个SkyAPM-dotnet 版本是1.3.0 。可以使用 https://www.nuget.org/packages/SkyAPM.Agent.AspNet 这个包试试看

**Reply by StevenWangGitHub** [reply]
这个我使用了，我把使用的dll文件手动考到发布工程的bin目录下，手动添加skyapm.json到项目目录下，重新IIS项目，交易不通，agent的日志也没有找到

**Reply by liuhaoyang** [reply]
添加这个包的依赖到项目里。需要完成的SkyAPM依赖。只拷贝这个dll运行不起来

**Reply by StevenWangGitHub** [reply]
![image](https://user-images.githubusercontent.com/30906238/223055212-2f1a5eea-9f67-4d7b-b145-8fed24ad4c8e.png)
我把所有的dll都拷贝了

**Reply by StevenWangGitHub** [reply]
我的操作步骤，是否有问题，是否有SkyAPM.Agent.AspNet部署手册？：
1、dotnet new mvc -n sampleapp（目的使用）
2、cd sampleapp
3、dotnet add package SkyAPM.Agent.AspNet
4、拷贝bin目录下所有的dll文件到发布工程的bin目录下
5、发布工程目录下添加skyapm.json
6、重启IIS asp.net application（.net framewolk 4.6.1）
7、交易不通，agent的日志也没有找到

**Reply by liuhaoyang** [reply]
> ![image](https://user-images.githubusercontent.com/30906238/223055212-2f1a5eea-9f67-4d7b-b145-8fed24ad4c8e.png) 我把所有的dll都拷贝了

这种方式没测试过

**Reply by StevenWangGitHub** [reply]
好的，谢谢

**Reply by StevenWangGitHub** [reply]
Microsoft.Extensions.Logging.Abstractions.dll 版本V1.1.1和V3.1.0冲突，删除V1.1.1就可以了
